partcad/src/partcad/exception.py

```
@@ -10,3 +10,11 @@
 
 class NeedsUpdateException(Exception):
     pass
+
+class PartFactoryBrepError(Exception):
+    """Custom exception for PartFactoryBrep errors."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)